apps/job-core/src/amqp.js

```diff
@@ -14,8 +15,13 @@ async function connect() {
 let channel
 export async function getAmqpChannel() {
   if (!channel) {
-    const connection = await connect()
-    channel = await connection.createChannel()
+    if (config.get('amqp.url').startsWith('redis://')) {
+      channel = buildAMQPToRedis(config.get('amqp.url'))
```

Comment:
Does it really make sense to re-use this configuration parameter?  What about `mq.type` (default `ampq`) and `redis.url`?  Are there technical challenges to doing this?

Reply:
Waiting a decision from @gregberge. If we fully move t Redis, tes we will improve the config